src/app/components/client/csat_survey/surveys/automaticRemovalCsatSurvey.ts

```diff
@@ -87,6 +87,20 @@ const surveyData: SurveyData = {
         "very-satisfied": "https://survey.alchemer.com/s3/7718562/002e20b6b82f",
       },
     },
+    {
+      id: "24-months",
+      showForUser: ["plus-user"],
+      showOnTab: ["fixed"],
+      daysThreshold: 716,
+      followUpSurveyOptions: {
+        "very-dissatisfied":
+          "https://survey.alchemer.com/s3/8176616/edb0c3f18778",
+        dissatisfied: "https://survey.alchemer.com/s3/8176616/9b6ca1d007a5",
+        neutral: "https://survey.alchemer.com/s3/8176616/37955cef0c5e",
```

Comment:
Your PR description says this should follow the pattern `7714663`, but I'm seeing the value mentioned here: `8176616`. I'm assuming that this is correct.